src/main/java/com/liveramp/hyperminhash/HmhCardinalityEstimator.java

```diff
@@ -1005,11 +1005,14 @@ static double biasCorrectEstimate(
     final int smallestDiffIndex = indexesByDifference.get(minDiff);
     final int secondSmallestDiffIndex = indexesByDifference.get(secondSmallestDiff);
 
+    int firstIndex = Math.min(smallestDiffIndex, secondSmallestDiffIndex);
```

Comment:
change that's worth calling out: sometimes the smallestDiffIndex was the higher x value, which broke the interpolator (it wants its input to be monotonically increasing on the X axis), so i made this change to always provide stuff in the right order. This assumes every array in rawEstimates is monotonically increasing (which they are)